pvacseq/lib/main.py

```diff
@@ -245,14 +267,20 @@ def main(args_input = sys.argv[1:]):
         sys.exit("No output files were created. Aborting.")
 
     combined_parsed_path      = combined_parsed_outputs(args, split_parsed_output_files, output_dir)
-    filt_out_path             = binding_filter(args, combined_parsed_path, output_dir)
+    final_path                = binding_filter(args, combined_parsed_path, output_dir)
+
+    if args.net_chop_method:
```

Comment:
Personally I prefer this form because I think it's simpler to look at, but if you like I can certainly change it